js/mexc.js

```diff
@@ -56,6 +64,7 @@ module.exports = class mexc extends Exchange {
                 'fetchWithdrawals': true,
                 'reduceMargin': true,
                 'setLeverage': true,
+                'setMarginMode': false,
```

Comment:
Does mexc only do cross?

Reply:
Mexc supports cross and isolated, I couldn't identify a method to switch the margin type. The user can choose the margin type with the parameter openType in createSwapOrder

Reply:
ok, sounds good